cex-collection/go-example/main.go

```diff
@@ -162,7 +168,7 @@ func cexDoGaslessTransfer() {
 
 	if sponsorableInfo.Sponsorable {
 		// Send the transaction using PaymasterClient
-		_, err := paymasterClient.SendRawTransaction(context.Background(), txInput)
+		_, err = privatePaymasterClient.SendRawTransaction(context.Background(), txInput, &paymasterclient.TransactionOptions{UserAgent: "MegaFuel/v1.2.2"})
```

Comment:
for cex, they dont need `UserAgent`, please use nil here.

Reply:
set it to nil